spec/acceptance/cs_location_spec.rb

```diff
@@ -98,7 +98,7 @@ class { 'corosync':
       expect(r.stdout).to match(/rsc_location/)
       # Feature not supported in Ubuntu
       expect(r.stdout).to match(/resource-discovery="exclusive"/) if fact('osfamily') == 'RedHat'
-      expect(r.stdout).to_not match(/resource-discovery="exclusive"/) if fact('osfamily') != 'RedHat'
+      expect(r.stdout).not_to match(/resource-discovery="exclusive"/) if fact('osfamily') != 'RedHat'
```

Comment:
is this new, or did this never work? or is it just RSpec rubocop saying we should do it?